core/src/test/java/org/infinispan/scattered/ControlledScatteredVersionManager.java

```diff
@@ -28,8 +28,14 @@ public static void flush(boolean expectRemoval, ControlledScatteredVersionManage
       for (;;) {
          boolean done = true;
          for (int i = 0; i < svms.length; ++i) {
-            if (regularCounters[i] >= svms[i].regularCounter.get()) done = false;
-            if (expectRemoval && removeCounters[i] >= svms[i].removeCounter.get()) done = false;
+            if (regularCounters[i] >= svms[i].regularCounter.get()) {
```

Comment:
Maybe we should put `svms[i].regularCounter.get()` into a field variable? Same with `removeCounter` below? Or are these cases not standard?

Reply:
OK, though this is just a test class...

Reply:
Sorry, overlooked this was in src/test, you can ignore.